coriolis/tests/test_data_transfer.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+import stat
+import struct
+
+import requests
+import requests_unixsocket
+
+from unittest import mock
+from urllib import parse
```

Comment:
These two imports should be grouped in the first group (where stdlib modules go). Please move these under `import struct`